example/fragment/gen/client.go

```diff
@@ -0,0 +1,93 @@
+// Code generated by github.com/Yamashou/gqlgenc, DO NOT EDIT.
+
+package gen
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/Yamashou/gqlgenc/clientv2"
+)
+
+type Client struct {
+	Client *clientv2.Client
+}
+
+func NewClient(cli *http.Client, baseURL string, interceptors ...clientv2.RequestInterceptor) *Client {
+	return &Client{Client: clientv2.NewClient(cli, baseURL, interceptors...)}
+}
+
+type Query struct {
+	Conversation *Conversation "json:\"conversation\" graphql:\"conversation\""
+}
+type Mutation struct {
+	Something string "json:\"something\" graphql:\"something\""
+}
+type AnswerFragment struct {
+	Answer *struct {
+		SelectionID string "json:\"selectionId\" graphql:\"selectionId\""
+	} "json:\"answer\" graphql:\"answer\""
+}
+type QuestionFragment struct {
+	Question *struct {
+		Options []*struct {
+			ID   string "json:\"id\" graphql:\"id\""
+			Text string "json:\"text\" graphql:\"text\""
+		} "json:\"options\" graphql:\"options\""
+	} "json:\"question\" graphql:\"question\""
+}
+type ConversationByIDQuery struct {
+	Conversation *struct {
+		Messages []*struct {
+			Answer *struct {
```

Comment:
@adam-zethraeus is this how you want the generated code to look like?

Reply:
Yes! i think that's a substantive improvement. At least now we can test the payload :).